CONTRIBUTING.md

```diff
@@ -16,12 +14,6 @@ By removing the assignment, we create space for other community members to take
 
 ## Sending PRs
 
-**OpenDAL's PR Roundtrip: 24 hours**
-
-OpenDAL promises fast iteration (I mean very fast).
-
-We commit to reviewing your PRs within 24 hours, and often within 8 hours. In return, we expect you to respond to our review within 24 hours as well. *It doesn't mean you MUST respond to us within 24 hours. You can contribute at your own pace.*
-
-If a PR receives no response for 24 hours, it will be closed. Please note that closing a PR does not mean rejection; we simply aim to keep our community iterating quickly by only tracking PRs awaiting review. By closing on-going (instead of on-reviewing) PR allow both of us foucing our own work.
+We will try our best to review your PRs within 24 hours, and often within 8 hours.
 
-You are welcome to reopen the PR at any time when you are ready for a new review.
+If a PR does not receive a response within 24 hours, it will be converted to draft status. To indicate that you are ready for review, please mark the PR as `ready for review`. By default, all reviewers ignore draft PRs.
```

Comment:
Maybe we could remove the last sentence. We don't need to leave this uncertainty behind.

Reply:
DONE.